java/src/main/java/com/adhoc/flight/client/BaseTinyIntVectorProcessor.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (C) 2017-2021 Dremio Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.adhoc.flight.client;
+
+public abstract class BaseTinyIntVectorProcessor implements TinyIntVectorProcessor {
```

Comment:
You can move the static methods into the interface instead of having this abstract class.

Reply:
Solved in the new version.